src/GHC/Util.hs

```diff
@@ -51,3 +59,72 @@ parseFileGhcLib :: FilePath -> String -> DynFlags -> ParseResult (Located (HsMod
 parseFileGhcLib filename str flags =
   GhclibParserEx.parseFile filename flags
     (if takeExtension filename /= ".lhs" then str else unlit filename str)
+
+{-# COMPLETE SrcSpan #-}
+-- | The \"Lin\" thing is because there is already e.g. 'SrcLoc.srcSpanStartLine'
```

Comment:
Typically we've been adding '. That convention has served us well, so I'd follow that.

Reply:
I originally didn't follow it because usually if `X` exists for HSE then `X'` is "the same thing" but for ghc. (while in this case `srcSpanStartLine` exists in ghc, not in HSE)

I guess this makes sense - I changed it.